modules/model/shared/src/test/scala/observe/model/ModelSuite.scala

```diff
@@ -7,6 +7,7 @@ import cats.kernel.laws.discipline.*
 import lucuma.core.util.arb.ArbEnumerated.given
 import observe.model.GmosParameters.*
 import observe.model.ObserveModelArbitraries.given
+import observe.model.ObserveModelArbitraries.given_Arbitrary_Operator
```

Comment:
Isn't this import included in the previous one?

Reply:
Ups